libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h

```diff
@@ -18,25 +18,35 @@
  */
 #define UART_BUSADDR                0x7E215000
 
-#define UART_PADDR_0                (UART_BUSADDR-BUS_ADDR_OFFSET+PADDDR_OFFSET)
+#define UART_PADDR_1                (UART_BUSADDR-BUS_ADDR_OFFSET+PADDDR_OFFSET)
```

Comment:
Stragen that the style checker does not complain about the whitespaces here. Could you add them to imprve readability?
```suggestion
#define UART_PADDR_1                (UART_BUSADDR - BUS_ADDR_OFFSET + PADDDR_OFFSET)
```

Reply:
should be fixed now